Rakefile

```diff
@@ -4,37 +4,23 @@ rescue LoadError
   puts 'You must `gem install bundler` and `bundle install` to run rake tasks'
 end
 
-require 'rdoc/task'
```

Comment:
The changes to this Rakefile are mostly superficial -- aligning it with the structure used by some of our other gems.